src/aeonlib/ocs/target_models.py

```diff
@@ -13,44 +13,34 @@ class SiderealTarget(BaseModel):
     model_config = ConfigDict(validate_assignment=True)
     name: Annotated[str, StringConstraints(max_length=50)] = "string"
     """The name of this Target"""
-    type: Literal["ICRS", "HOUR_ANGLE"]
+    type: Literal["ICRS", "HOUR_ANGLE", "ALTAZ"]
     """The type of this Target"""
-    hour_angle: float | None = None
-    """Hour angle of this Target"""
+    hour_angle: Annotated[float, Ge(-180), Le(180)] | None = None
```

Comment:
Realistically this is normally written in HH:MM:SS.ss but much like RA, people would probably need to do that conversion before submission.

Reply:
Could it be an Angle?
https://docs.astropy.org/en/stable/api/astropy.coordinates.Angle.html

Reply:
absolutely. That is the correct answer.
Everything else here should be an `Angle` as well, but I didn't know how easy that would be to implement.